dm_regional_app/charts.py

```diff
@@ -218,11 +221,27 @@ def prediction_chart(historic_data: PopulationStats, prediction: Prediction, **k
     # Visualise prediction using unstacked dataframe
     fig = go.Figure()
 
+    # Append graph info to population data dictionaries
+    historic_care_by_type_dfs["type"] = "Historic"
+    historic_care_by_type_dfs["dash"] = "dot"
+
+    forecast_care_by_type_dfs["type"] = "Base forecast"
+    forecast_care_by_type_dfs["dash"] = None
+
+    # Add dictionary to list to be added to graph
+    traces_list = [historic_care_by_type_dfs, forecast_care_by_type_dfs]
+
     # Add forecast and historical traces
-    fig = add_traces(forecast_care_by_type_dfs, historic_care_by_type_dfs, fig)
+    fig = add_traces(fig, traces_list)
```

Comment:
This is up to personal preference. I usually prefer not to define variables if they aren't being re-used. So here I would opt for:
```suggestion
    fig = add_traces(fig, [historic_care_by_type_dfs, forecast_care_by_type_dfs])
```
The idea being that creating a variable could imply the need for re-use which adds to developer mental load when reading/modifying the code. [This readme](https://github.com/zakirullin/cognitive-load?tab=readme-ov-file#cognitive-load) has some interesting other practices around reducing this 🙂

Reply:
I can get behind that idea - thanks!